docs/part-3-serve-and-deploy-the-model/chapter-14-deploy-and-access-the-model-on-kubernetes.md

```diff
@@ -103,9 +103,37 @@ flowchart TB
 
 ## Steps
 
+### Install the Kubernetes CLI
+
+Install the Kubernetes CLI (kubectl) on your machine.
+
+=== ":simple-googlecloud: Google Cloud"
+
+    Install kubectl with the Google Cloud CLI:
+
+    ```sh title="Execute the following command(s) in a terminal"
+    # Install kubectl with gcloud
+    gcloud components install kubectl
```

Comment:
This seemingly didn't work on Ubuntu - do we add a note to alternatively install kubectl with the distro package manager if an error/warning occurs here?